src/components/common/header/header.scss

```diff
@@ -163,7 +163,26 @@ header {
     }
```

Comment:
Check mobile. It's not showing up in the mobile menu.

Reply:
Also, let's have the attention hierarchy secondary or tertiary.

So right now these get the most attention right now:
<img width="394" alt="Screen Shot 2020-10-09 at 5 30 05 PM" src="https://user-images.githubusercontent.com/14047327/95641080-13771000-0a55-11eb-876c-4182fc387fc8.png">
<img width="144" alt="Screen Shot 2020-10-09 at 5 30 46 PM" src="https://user-images.githubusercontent.com/14047327/95641095-2b4e9400-0a55-11eb-9e9b-19f7c8a51217.png">

I think just purple text with purple underline would be good and simple.